### PR TITLE
Donate/Pay a user tutorial

### PR DESCRIPTION
**Pull request by leandrogavidia**
This PR adds a step-by-step tutorial on how to create an action for a donation scenario.

### CODE REVIEW

File: TUTORIAL.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+Solana Actions are APIs that adhere to the Solana Actions Specification. Our Solana Actions must be capable of returning:
+
+1. Action Metadata
+2. A transaction to be sent to the Solana blockchain.
+
+## Let's Start!
+
+First, we will create our workspace using `znap-cli`. If you do not have `znap-cli` installed, follow our guide [here](https://github.com/heavy-duty/znap/blob/master/INSTALLATION.md).
+
+Next, let's create a new project with `znap init <project-name>`.
```
Comment (danmt) [inline TUTORIAL.md:22]: I'd like to include some information about what happens when `init` is called, which files are created, where, etc... similar to the case of `new` in step 1.

File: TUTORIAL.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+### 1. Let's create our action collection for Alice's campaign
+
+In the root directory of your project, run the following command to create a collection: `znap new alice_campaign`. This command will create a new action collection called 'alice_campaign' inside the `/collections` folder, where we will place our actions for Alice.
```
Comment (danmt) [inline TUTORIAL.md:26]: This command should be called with snake case `alice-campaign`. The name of the collection's folder also should be with snake case.

The exception here is the `pub mod alice_campaign` whish should be using the underscore notation.

**Review by danmt — CHANGES_REQUESTED** [review]
after the requested changes, this should be ready to go.

File: TUTORIAL.md (hunk 2 above)
Comment (leandrogavidia) [reply TUTORIAL.md]: Done

File: TUTORIAL.md (hunk 1 above)
Comment (leandrogavidia) [reply TUTORIAL.md:22]: Done

**Review by danmt — APPROVED** [review]

**Comment by danmt** [comment]
LGTM

**Pull request merged**